envConfig/local.env

```diff
@@ -2,6 +2,7 @@ SIMORGH_BASE_URL=http://localhost:7080
 SIMORGH_ASSETS_MANIFEST_PATH=build/assets-local.json
 SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN=http://localhost:7080
 SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/
+MPULSE_API_KEY=JLSEX-R76DK-JXL3A-UB7YH-NPSEW
```

Comment:
Did you see Simon's comment on Slack about externalising this?

Reply:
So this is still adding the key as an environment variable. Or do you think we should move it to something like cosmos config? Which will still be committed to a repo.

Reply:
I think we would still want a local env var, although I suppose that could just be a placeholder (XXXXX-XXXXX etc). Probably would make sense to use cosmos for the test and live env.